pagination/keysetpagination/paginator.go

```diff
@@ -66,8 +66,7 @@ func Paginate[I Item](p *Paginator) pop.ScopeFunc {
 		eid := q.Connection.Dialect.Quote(id)
 		return q.
 			Limit(p.Size()+1).
-			Where(fmt.Sprintf(`%s > ?`, eid), p.Token()).
-			Order(fmt.Sprintf(`%s ASC`, eid))
```

Comment:
If no order is specified in the query, this has to fall back to the primary ID for ordering. How can we address that?

Reply:
Hm, we would need to add ordering to the pagination params, then we could do "if order not supplied, ORDER BY PK".

But why does it need to do that? I feel "if the query is not ordered, let the DB decide" is fine, no?

Reply:
I added optional ordering to the paginator, PTAL. It falls back to the PK if no ordering is supplied.